src/core/mapping/base_mapper.cc

```diff
@@ -963,8 +973,8 @@ void BaseMapper::legate_select_sources(const MapperContext ctx,
     } else
       band_ranking.push_back(std::pair<PhysicalInstance, uint32_t>(instance, finder->second));
   }
-  assert(!band_ranking.empty());
-  // Easy case of only one instance
+  // assert(!band_ranking.empty());
```

Comment:
remove these bogus changes

Reply:
the ` assert(!band_ranking.empty());` doesn't work anymore when all instances are collective. Do you think I need to modify logic for legate_select_sources` to parse thought collectiveViews as well?